core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4611,6 +4611,14 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.IGNORE)
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey FUSE_UMOUNT_WAITTIME =
+      new Builder(Name.FUSE_UMOUNT_WAITTIME)
+          .setDefaultValue("1min")
+          .setDescription("The maximum wait time to wait for all file in stream and out stream "
+              + "closed in Fuse umount() operation.")
```

Comment:
Could you describe what happens after this waiting period?

Reply:
Added the description, PTAL